src/ascii_string.rs

```diff
@@ -566,7 +566,6 @@ pub trait IntoAsciiString<T: ?Sized+AsciiExt<Owned=Self>> : Sized+Borrow<T> {
     }
 }
 
-#[cfg(feature = "unstable")]
 impl IntoAsciiString<AsciiStr> for AsciiString {
     fn into_ascii(self) -> Result<AsciiString,AsciiString> {
```

Comment:
Like above I'd like to name this `into_ascii_string`. It's a bit longer but also more precise when used in code with inferred types.